qcodes/station.py

```diff
@@ -217,3 +251,214 @@ def __getitem__(self, key):
         return self.components[key]
 
     delegate_attr_dicts = ['components']
+
+    def load_config_file(self, filename: Optional[str] = None):
+        def get_config_file_path(
+                filename: Optional[str] = None) -> Optional[str]:
+            filename = filename or get_config_default_file()
+            if filename is None:
+                return None
+            search_list = [filename]
+            if (not os.path.isabs(filename) and
+                get_config_default_folder() is not None):
```

Comment:
Sorry, that is hard to see in the preview from Github: do you want to align `get_config_default_folder` with `os....`?

Reply:
yes, indeed :)

Reply:
why would you prefer it there? (The not is only referring to the `isabs` and not to  `get_config_default_folder`.)

Reply:
because of separating the "expression" of the "if" from the "body" of the "if".

please feel free to choose a way to do it, but i'd like to avoid the "blend" or the "if" "expression" and the "if" "body".

Reply:
Ah, because it coincides with the indent of the body? That is a good point. It would improve readability if it had a different indent. Great! I just found the style guide for it: https://www.python.org/dev/peps/pep-0008/#multiline-if-statements
Thanks for teaching me this Mikhail!

Reply:
:)